application/deed/utils.py

```diff
@@ -29,4 +30,20 @@ def _create_title_validator():
     return validator(schema)
 
 
+def validation_checks(result, borrower):
+    try:
+        if result:
+            return result
+
+        borrower_date = datetime.datetime.strptime(
+            borrower['dob'], '%d/%m/%Y')
+
+        if borrower_date > datetime.datetime.now():
+            return True
+
+        return False
+    except:
+        return True
```

Comment:
If it throws an exception then its valid? 

Its not clear from reading it what sort of validation this is